bootstrap/bootstrap.go

```diff
@@ -858,12 +858,12 @@ func (b *Bootstrap) executePluginHook(ctx context.Context, name string, checkout
 		envMap, err := p.ConfigurationToEnvironment()
 		if dnerr := (&plugin.DeprecatedNameErrors{}); err != nil && errors.As(err, &dnerr) {
 			b.shell.Logger.Headerf("Deprecated environment variables for plugin %s", p.Plugin.Name())
-			b.shell.Logger.Printf(
-				"The way that environment variables are derived from the plugin configuration is changing. " +
-					"We'll export both the deprecated and the replacement names for now. " +
-					"You may be able avoid this by removing consecutive underscore, hypen, or whitespace " +
-					"characters in your plugin configuration.",
-			)
+			b.shell.Logger.Printf("%s", strings.Join([]string{
```

Comment:
Just to understand, I'm guessing the problem with using a single string literal is how long it would be?

Reply:
Yep. Wrapped lines are irritating for many reasons